l10n_it_central_journal/__manifest__.py

```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Author: Gianmarco Conte - Dinamiche Aziendali Srl
-# Copyright 2017
+# Copyright 2018
 # Dinamiche Aziendali Srl <www.dinamicheaziendali.it>
```

Comment:
Please fix copyrights following the format defined in the [template module](https://github.com/OCA/maintainer-tools/tree/master/template/module): generic in `__init__` files, with authors elsewhere (xml files too)